docs/workflows/api-token.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: API Tokens
-sidebar_position: 6
+sidebar_position: 4
```

Comment:
The Realms and Application guides link to this tokens guide in the creating them by API sections. Probably fine to link this guide after them though.

Reply:
How does that impact numbering?

Reply:
The API Tokens guide is displayed after the Realms and Application guides based on numbering

Reply:
Oh I see. What do you think a good ordering would be?

Reply:
I'm not sure. I think what you have is probably fine. I was just pointing out that the API Token workflow would be listed after 2 guides that link to with this new order, but it's probably fine to do that.

Reply:
The ordering here is identical to what it currently is in production, except for the bind and authenticate guides. Do you think we should move API tokens to after account setup?

Reply:
I think it's fine either way. I'll resolve this so you can merge.